lib/messages/pull-request.js

```diff
@@ -109,29 +126,25 @@ class PullRequest extends AbstractIssue {
     const attachments = [
       this.getMainAttachment(),
     ];
-    if (this.statuses) {
-      // 3 cases: none or one successful, some successful, all successful
```

Comment:
Do you reckon it would be useful to keep one summary comment at the top of this logic like the one we had before?